snorkel/learning/disc_learning.py

```diff
@@ -42,200 +37,85 @@ def score(self, session, X_test, test_labels, gold_candidate_set=None, b=0.5, se
         return s.score(test_marginals, train_marginals, b=b,
```

Comment:
I'm fine just getting rid of this, whatever is fast & cleans this loose thread up